test/edu-benefits/pages/schoolSelection.unit.spec.jsx

```diff
@@ -1,141 +1,76 @@
+import _ from 'lodash/fp';
+
 import React from 'react';
 import { findDOMNode } from 'react-dom';
 import { expect } from 'chai';
 import sinon from 'sinon';
 import ReactTestUtils from 'react-addons-test-utils';
 
 import { DefinitionTester, submitForm } from '../../util/schemaform-utils.jsx';
-import formConfig from '../../../src/js/edu-benefits/pages/schoolSelection.js';
```

Comment:
I'm using the config from the form instead of the config from the page because 1) it seemed like we were testing the forms more than the page anyhow, and 2) I ran across an instance where, after refactoring, I called the page building function with a bad parameter. The test ran fine, so I thought it was golden, but when I went to take a screenshot for this PR, the page was completely blank. No JS errors, just blank.